Tutorials/BoseHubbard1d/bosehubbard1d.py

```diff
@@ -12,67 +12,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import netket as nk
+from mpi4py import MPI
```

Comment:
Unrelated to this specifically, but is it possible to automatically import MPI when we import netket? I guess so, right ?

Reply:
We don't really need to _import_ anything, we'd just like to call `MPI_Init`. It's on my TODO list to implement it -- should be trivial.